### ISSUE TITLE
cmake failing with boost 1.71

### ISSUE THREAD

**Issue by nikisalli** [CCNYRoboticsLab/imu_tools#100]
on Arch linux this package only compiles when using a custom boost 1.68 installation.

**Comment by jhbruhn** [comment]
This does compile when doing the following change:

```diff
index c41261b..ab43642 100644
--- a/imu_filter_madgwick/CMakeLists.txt
+++ b/imu_filter_madgwick/CMakeLists.txt
@@ -3,8 +3,8 @@ project(imu_filter_madgwick)
 
 find_package(catkin REQUIRED COMPONENTS roscpp sensor_msgs geometry_msgs tf2 tf2_geometry_msgs tf2_ros nodelet pluginlib message_filters dynamic_reconfigure)
 
-find_package(Boost REQUIRED COMPONENTS system thread signals)
-
+#find_package(Boost REQUIRED COMPONENTS system thread signals)
+find_package(Boost)
 # Generate dynamic parameters
 generate_dynamic_reconfigure_options(cfg/ImuFilterMadgwick.cfg)

```

I have not tested how backwards compatible this is.

**Comment by mintar** [comment]
This was fixed by #103 .
